front-end/src/components/WorkoutsForm.js

```diff
@@ -0,0 +1,50 @@
+import React, { Component } from 'react';
+
+export default class WorkoutsForm extends Component {
+  state = {
+    title: '',
+    name: '',
+    weight: '',
+    sets: '',
+    reps: ''
+  };
+
+  changeHandler = e => {
+    this.setState({ [e.target.name]: e.target.value });
+  };
+
+  submitHandler = e => {
+    e.preventDefault();
+    const newWorkout = {
+      title: this.state.title,
+      name: this.state.name,
+      weight: this.state.weight,
+      sets: this.state.sets,
+      reps: this.state.reps
+    };
+    //this.props.postWorkout(newWorkout) <-- Functionally will be added soon
+  };
+
+  render() {
+    return (
+      <div className="workouts-form">
+        <form onSubmit={this.submitHandler}>
+          <h5>Workout Creator:</h5>
+          <input type="text" name="title" placeholder="Title" />
+          <select>
+            <option value="category">Arms</option>
```

Comment:
Consider using map for the options and inputs here as well

Reply:
Good idea. I maayyy not do it on this pull. Next week I plan on making this component fully functional with the back-end and also responsive.